pkg/storaged/dialog.jsx

```diff
@@ -214,10 +214,11 @@
 import cockpit from "cockpit";
 
 import React from "react";
-import { OverlayTrigger, Tooltip } from "patternfly-react";
+import { OverlayTrigger, Tooltip, TypeAheadSelect } from "patternfly-react";
 
 import { show_modal_dialog } from "cockpit-components-dialog.jsx";
 import { StatelessSelect, SelectEntry } from "cockpit-components-select.jsx";
+
```

Comment:
Not related newline change.

Reply:
damn, should I repush? (I just like how green the tests are - even selenium ones :D )